src/lib.rs

```diff
@@ -2347,10 +2348,11 @@ pub unsafe trait FromBytes: FromZeros {
 
     /// Interprets the prefix of the given `bytes` as a `&Self` without copying.
     ///
-    /// This method returns both a reference to the first `size_of::<Self>()`
-    /// bytes of `bytes` interpreted as a `Self`, and a reference to the remaining
-    /// bytes. If `bytes.len() < size_of::<Self>()` or `bytes` is not aligned to
-    /// `align_of::<Self>()`, this returns `Err`.
+    /// This method computes the largest possible size of `Self` that can fit in
+    /// the leading bytes of `candidate`, then attempts to return both a
```

Comment:
Whoops, this shouldn't be "bytes of `candidate`.

"bytes of `bytes`" is awkward, so perhaps I'll rename `bytes` to `source`.

Reply:
Done.